project/pom.xml

```diff
@@ -72,7 +72,30 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>23.0</version>
+      <version>29.0-jre</version>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/com.google.api-client/google-api-client -->
+    <dependency>
+      <groupId>com.google.api-client</groupId>
+      <artifactId>google-api-client</artifactId>
+      <version>1.23.0</version>
+      <exclusions>
```

Comment:
Did you exclude Guava since you already have a dependency on it?

Reply:
Yes. The double dependency caused issues and made other parts of the code to stop working.

Reply:
Thanks for explaining!